packages/web/src/scss/components/Checkbox/_Checkbox.scss

```diff
@@ -2,6 +2,7 @@
 //    and therefore bloating the CSS.
 // 2. We need to set the background-color of the non-disabled Checkbox input to initial color when it is hovered
 //    or active, because we need it contrasting with the background color.
+// 3. Make the text selectable
```

Comment:
Please add that it also allows clicking on any interactive elements inside, for example links.